### PR TITLE
Network e2e test, vms/pods connectivity 

### PR DESCRIPTION
**Pull request by shiywang**

### CODE REVIEW

**Comment by gbenhaim** [comment]
ci test please

**Comment by shiywang** [comment]
@nellyc hi, I think @duyanyan has no problem with this automation, do you think there's someone else need to help review this pr ?

File: tests/network/network_connectivity_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		vmbIP = vmb.Status.Interfaces[0].IP
+	})
+
+	Context("Connectivity between VMs", func() {
+
+		It("in vmb ping vma should be successful", func() {
+			expecter, err := tests.LoggedInCirrosExpecter(vmb)
+			Expect(err).ToNot(HaveOccurred())
+			defer expecter.Close()
+			cmdCheck := fmt.Sprintf("ping -w 3 %s \n", vmaIP)
```
Comment (gbenhaim) [inline tests/network/network_connectivity_test.go]: Why not check the return code?

File: tests/network/network_connectivity_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Network Connectivity", func() {
+
+	flag.Parse()
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	var vma *v1.VirtualMachineInstance
```
Comment (gbenhaim) [inline tests/network/network_connectivity_test.go]: Please use arrays to store the VMs structs, and then replace the duplicate code with loops.

File: tests/network/network_connectivity_test.go (hunk 1 above)
Comment (shiywang) [reply tests/network/network_connectivity_test.go]: Okay, I check the message `3 packets transmitted` so even if there's some package lost due to networking latency or sth(not 0% package loss), it still will be able to send out the package. is that Okay ?

**Comment by shiywang** [comment]
@gbenhaim updated, PTAL

**Comment by shiywang** [comment]
@gbenhaim PTAL, thanks

**Pull request merged**